maintainers/maintainer-list.nix

```
@@ -10774,4 +10774,14 @@
     github = "benneti";
     githubId = 11725645;
   };
+  dschrempf= {
```

Please put this into a separate commit with the message ``maintainers: add dschrempf``.

```suggestion
  dschrempf = {
```